pyOCD/target/target_kinetis.py

```diff
@@ -141,3 +164,18 @@ def massErase(self):
         else:
             logging.error("Failed to unlock. MDM status: 0x%x", val)
             return False
+
+    ## @brief Write the unsecure FSEC value to flash.
+    #
+    # This will prevent the device from being secured after the next reset.
+    def unlockFlash(self):
+        logging.info("Unlocking chip...")
+
+        logging.info("Performing mass erase")
+        self.massErase()
+
+        # Write FCF
+        logging.info("Writing FCF = %s" % repr(fcf))
+        self.flash.init()
+        self.flash.programPage(FCF_ADDR, fcf)
```

Comment:
Do you plan on keeping this?  According to AN4835, a mass erase triggered by the MDM-AP will program the FSEC to unsecure (0xFE), so this is unnecessary.  Just wanted to make sure you got my last comment on #135.